test/activation_based/test_functional_neuron.py

```
@@ -281,6 +415,57 @@ def test_lava_cuba_lif_step_matches_norm_free_module():
+    x_seq = torch.randn(3, 2, 4)
+    states = tuple(module._memories.values())
+
+    outputs, updated_states = module.functional_forward((x_seq,), states)
+
+    assert module.v_seq is None
+    assert module.i_seq is None
+    assert tuple(module._memories.values()) == states
+    torch.testing.assert_close(module(x_seq), outputs[0])
+    assert module.v_seq.shape == x_seq.shape
+    assert module.i_seq.shape == x_seq.shape
+    for actual, expected in zip(module._memories.values(), updated_states):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)